CHANGELOG.md

```diff
@@ -3,71 +3,129 @@
+- Fix a possible deadlock on closing the watcher on kqueue [moby/moby#37392](https://github.com/moby/moby/pull/37392)
+- Use poller based watcher to work around the file caching issue in Windows [moby/moby#37412](https://github.com/moby/moby/pull/37412)
+
 
 ### Networking
 
++ Add support for global default address pools [moby/moby#37558](https://github.com/moby/moby/pull/37558) [docker/cli#1233](https://github.com/docker/cli/pull/1233)
+* Use direct server return (DSR) in east-west overlay load balancing [docker/engine#93](https://github.com/docker/engine/pull/93) / [docker/libnetwork#2270](https://github.com/docker/libnetwork/pull/2270)
 * Builder: temporarily disable bridge networking when using buildkit. [moby/moby#37691](https://github.com/moby/moby/pull/37691)
-* Move network conversions out of API router. [moby/moby#37156](https://github.com/moby/moby/pull/37156)
-* Use sortorder library for sorting network list output. [docker/cli#1266](https://github.com/docker/cli/pull/1266)
-- Fix faulty error type checking in removeNetwork(). [moby/moby#37409](https://github.com/moby/moby/pull/37409)
+- Handle systemd-resolved case by providing appropriate resolv.conf to networking layer [moby/moby#37485](https://github.com/moby/moby/pull/37485)
+
 
 ### Runtime
 
-* Disable TestExecWindowsOpenHandles on RS5 temporarily. [moby/moby#37679](https://github.com/moby/moby/pull/37679)
-* LCOW: Ensure platform is populated on COPY/ADD. [moby/moby#37563](https://github.com/moby/moby/pull/37563)
-* LCOW: Mount to short container paths to avoid command-line length limit. [moby/moby#37659](https://github.com/moby/moby/pull/37659)
-* LCOW: lazycontext: Use correct lstat, fix archive check. [moby/moby#37356](https://github.com/moby/moby/pull/37356)
-* Lcow: fix debug in startServiceVMIfNotRunning(). [moby/moby#37446](https://github.com/moby/moby/pull/37446)
-* Revendor Microsoft/opengcs @ v0.3.8. [moby/moby#37515](https://github.com/moby/moby/pull/37515)
-* Select polling based watcher for docker log file watcher on Windows. [moby/moby#37412](https://github.com/moby/moby/pull/37412)
-* Updated the go-winio library to release 0.4.8 that has the fix for Windows Container. [docker/cli#1157](https://github.com/docker/cli/pull/1157)
-+ Add --chown flag support for ADD/COPY commands for Windows. [moby/moby#35521](https://github.com/moby/moby/pull/35521)
-+ Adds LinuxMetadata support by default on Windows. [moby/moby#37514](https://github.com/moby/moby/pull/37514)
++ Configure containerd log-level to be the same as dockerd [moby/moby#37419](https://github.com/moby/moby/pull/37419)
++ Add configuration option for cri-containerd [moby/moby#37519](https://github.com/moby/moby/pull/37519)
++ Update containerd client to v1.2.0-rc.1 [moby/moby#37664](https://github.com/moby/moby/pull/37664), [docker/engine#75](https://github.com/docker/engine/pull/75) / [moby/moby#37710](https://github.com/moby/moby/pull/37710)
```

Comment:
We should also mention what version of Containerd we ship with. I wasn't sure what the exact containerd version is now that we bundle, because it doesn't match the "static" version that's in the Dockerfile. Is it v1.2.0 GA now?

@seemethere